tests/test_interface/test_interface.py

```diff
@@ -23,7 +23,7 @@ def _io_similar(lhs, rhs):
 
 
 def test_1():
-    I0 = m.DeclareInterface("a", m.In(m.Bit), "b", m.Out(m.Bits[2]))
```

Comment:
Should we keep the old function and deprecate it? Just in case there's external code using it

Reply:
Don't think anyone is using it but added it back.